script/build.js

```diff
@@ -89,9 +92,21 @@ smith.source(sourceDir);
 smith.destination(`../build/${options.buildtype}`);
 
 // Ignore files that aren't ready for production.
+//
+// Maintain as minimal a set of ignore files for the staging environment as
+// possible. The staging environment is simply the easiest workaround to allow
+// end to end testing of resources outside of our direct control. This becomes
+// an axis of divergance that may cause _major_ problems with the production
+// build that cannot be easily detected, so it should be used sparingly.
+//
+// Ideally, as soon as a feature has been tested, it should be removed from the
```

Comment:
I'm a little confused here. Is the idea that files in the `ignoreList` for `production` will all be available in staging _unless_ they are in the `ignoreList` for `staging`?